layouts/partials/comment/valine.html

```diff
@@ -0,0 +1,10 @@
+<div id="valine-comments" style="margin-top: 32px;"></div>
+<script src='//unpkg.com/valine/dist/Valine.min.js'></script>
+<script type="text/javascript">
+new Valine({
+    el: '#valine-comments' ,
+    appId: '{{ .Site.Params.comment.valine.appId }}',
+    appKey: '{{ .Site.Params.comment.valine.appKey }}',
+    placeholder: '{{ .Site.Params.comment.valine.placeholder }}'
```

Comment:
I found that there are many other options like placeholder in valine. So you can just iterate all the optional params in `params.yaml`. A similar approach for KaTex options can be found at  `head.html:119`.